websites/M/Mavanimes/presence.ts

```diff
@@ -0,0 +1,74 @@
+const mavanimes = new Presence({
+    clientId: "814986239681626143"
+  }),
+
+  video: {
+  dataAvailable?: boolean;
+  currentTime?: number;
+  duration?: number;
+  paused?: boolean;
+} = {};
+
+mavanimes.on("iFrameData", (videoData: {
+    error?: boolean;
+    currentTime: number;
+    duration: number;
+    paused: boolean;
+  }) => {
+  if (!videoData.error) {
+    video.dataAvailable = true;
+    video.currentTime = videoData.currentTime;
+    video.duration = videoData.duration;
+    video.paused = videoData.paused;
+  }
+});
+
+mavanimes.on("UpdateData", async () => {
+  const data: PresenceData = {
+    largeImageKey: "logo"
+  },
+    animeName = document.getElementsByClassName("entry-title")[0].innerHTML,
+    url = new URL(window.location.href),
+    params = new URLSearchParams(url.search);
+  
+  if (document.location.pathname === "/") {
+    if (params.has("s") === true) {
+      data.details = "Cherche un animé..";
+      data.state = params.get("s");
+    } else
+      data.details = "Page d'accueil";
+  } else if (document.location.pathname.endsWith("/tous-les-animes-en-vf/")) 
+    data.details = "Cherche un animé en VF..";
+  else if (document.location.pathname.endsWith("/films/")) 
+    data.details = "Cherche un film..";
+  else if (document.location.pathname.endsWith("/tous-les-animes-en-vostfr-fullhd-2/")) 
+    data.details = "Cherche un animé..";
+  else if (document.location.pathname.endsWith("/regarder-animes-oav-streaming/")) 
+    data.details = "Cherche un OAV..";
```

Comment:
doesn't use many dom elements, maybe display what the user is actually searching for

Reply:
I know but it works like that